web/src/app/lists/FlatList.tsx

```diff
@@ -202,16 +204,23 @@ export default function FlatList({
     return items.map((item, idx) => {
       if ('subHeader' in item) {
         return (
-          <CSSTransition key={idx} timeout={0} exit={false} enter={false}>
+          <CSSTransition
+            key={'header_' + item.id + idx}
+            timeout={0}
+            exit={false}
+            enter={false}
+          >
             {renderSubheaderItem(item, idx)}
           </CSSTransition>
         )
       }
       if ('type' in item) {
         return (
           <CSSTransition
-            key={'notice_' + (item.id || idx)}
+            key={'notice_' + item.id + idx}
             timeout={500}
+            exit={false || item.transition}
```

Comment:
This will always return what's after the `||`, to convert it to bool either use `Boolean(item.transition)` or `!!item.transition` alternatively swapping the order to `item.transition || false` would also give the intended effect of returning `false` for all falsy values.


example: `console.log(false || 0)` prints `0` not `false`.